### ISSUE TITLE
Add TypeScript definitions files

### ISSUE THREAD

**Issue by robregonm** [bitovi/canjs.com#180]
TypeScript is getting more popular nowadays, and it would be great to have the definitions files available in order to develop using TypeScript.
Take a look at: https://github.com/borisyankov/DefinitelyTyped

**Comment by robregonm** [comment]
http://developer.telerik.com/featured/the-rise-of-typescript/

**Comment by marknadig** [comment]
@robregonm did you ever find typings files for can?

**Comment by robregonm** [comment]
@marknadig No, I didn't find it, I tried to do it by myself many months ago, however since I don't know all the nuances and dependencies I never finished, so I moved to another framework.
